nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/ValidateRecord.java

```diff
@@ -166,6 +171,16 @@
         .required(true)
         .build();
 
+
+    static final PropertyDescriptor ATTRIBUTE_NAME_TO_STORE_FAILURE_DESCRIPTION = new PropertyDescriptor.Builder()
+        .name("emit-failure-description-property")
+        .displayName("Variable Describing Parse Failure")
```

Comment:
We should use the word 'attribute' here, rather than 'variable', as these have different meanings in the context of NiFi.